src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs

```diff
@@ -72,9 +72,17 @@ public void TestAutomationStartUnpublishedRunbook()
             TestRunner.RunTestScript("Test-AutomationStartUnpublishedRunbook -runbookPath ScenarioTests\\Resources\\Test-WorkFlowWithVariousParameters.ps1");
         }
 
-        [Fact(Skip = "Need x64 test framework.")]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        // Playback error:
+        /* Message: System.Management.Automation.ActionPreferenceStopException :
+         * The running command stopped because the preference variable "ErrorActionPreference" or common parameter
+         * is set to Stop: Unable to find a matching HTTP request for URL
+         * 'PUT /subscriptions/d2b38167-d3ca-4d1f-a020-948eee21b6bc/resourceGroups/to-delete-01/providers/Microsoft.Automation/automationAccounts/fbs-aa-01/jobs/bf922f22-eafa-4208-a6c7-55b5431a9da4?api-version=2017-05-15-preview'.
```

Comment:
@ikanni I may have sent the comment offline, but this error means that the HttpRecorder cannot find a request in the `.json` recordings for this test that matches the one being constructed by the cmdlet. Since this request URI contains a GUID, my hunch is that there is a randomly generated GUID somewhere in the `Import-AzAutomationRunbook` pipeline (since that's the only main difference between the old and new test), and this is causing the recorded GUID to not match up with the randomly generated one. We may need to update the PowerShell-based Automation Client to allow us to set GUIDs for tests to allow these recordings to pass.